### PR TITLE
Add idt.js script and related npm run scripts v2

### PR DESCRIPTION
**Pull request by gibfahn**
Continuation of https://github.com/ibm-developer/generator-ibm-core-node-express/pull/20

Still needs tests, will add shortly.

### CODE REVIEW

File: app/templates/idt.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+function runIDT(args) {
+  const cmd = 'bx dev ' + args.join(' ');
+  console.log(chalk.blue('Running:'), cmd);
+  cp.execSync(cmd, {stdio: 'inherit'});
+}
+
+// Download the IDT installer script and trigger run.
+function downloadInstaller(cb) {
+  const url = win ?
+    'http://ibm.biz/idt-win-installer' :
```
Comment (gibfahn) [inline app/templates/idt.js]: We should change this to the yeoman version of the link once available.

File: app/templates/idt.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+function runIDT(args) {
+  const cmd = 'bx dev ' + args.join(' ');
+  console.log(chalk.blue('Running:'), cmd);
+  cp.execSync(cmd, {stdio: 'inherit'});
+}
+
+// Download the IDT installer script and trigger runIDT().
+function downloadInstaller(cb) {
+  const url = win ?
+    'https://ibm.biz/yeoman-idt-win-install' :
```
Comment (gibfahn) [inline app/templates/idt.js]: This link isn't in place yet.

**Comment by gibfahn** [comment]
Added some docs in the README, PTAL.

@cvignola I think this is good to land from my end, let me know if something should be changed.

File: app/templates/idt.js (hunk 2 above)
Comment (cvignola) [reply app/templates/idt.js]: I know.  I just asked Emma again.  She's been busy.  It will be before this code is live.

**Comment by cvignola** [comment]
fyi, Adam's review was simply that we create the windows installer link, which we're doing today

**Pull request merged**